pkg/cmd/cli/describe/describer.go

```diff
@@ -946,6 +955,8 @@ func (d *TemplateDescriber) DescribeTemplate(template *templateapi.Template) (st
 		formatMeta(out, template.ObjectMeta)
 		out.Write([]byte("\n"))
 		out.Flush()
+		d.DescribePostProcessMessage(template.PostProcessMessage, out)
```

Comment:
the parameters seem more important than the post process message, and the message is likely to be longish, so I'd put it after the params